### ISSUE TITLE
Finding a balance between user-friendliness and yeetability

### ISSUE THREAD

**Issue by 93null** [93null/fubar#32]
# The ideal web interface
The ideal web interface would, out of the box, make at least the following obselete:

### Marketers
The ideal web interface does not display ads, nor support services that hide content behind ads.

### Subscription Services
The ideal web interface uses public web services to provide the same value as services like Spotify, Netflix, etc.

### Centralization
The ideal web interface supports decentralized alternatives to services provided by banks, social media, "gig-economy" middlemen leeches, etc.

### Search Engines
The ideal web interface enables and encourages users to create and share search indexes, search commands (url/search?q=%s), bookmarks/lists, etc. Unlike browsers, it fascilitates customization that favors the user over marketers, and does not promote low-quality services because it could not be incentivized to do such a thing.

### 3rd-Party Browsers
Fubar automates Edge, which comes with Windows. I don't dislike Edge more than other browsers but, for many people, the first app downloaded on a fresh installation of windows is another Browser. Fubar will be downloaded, installed, and ready to use faster than popular browsers, because it uses a browser that you already have and does not bombard you with prompts for telemetry, marketing, default browser, user accounts, etc..

### User Accounts
The ideal web interface does not provide links to solutions, but the solution itself. Searching for images will not return links to websites that might require authentication just to get a webp image - it will just return the images if available, and options to save/share/edit them. Additionally, it supports privacy-respecting technologies.

# The problem

Releasing the ideal web interface would, best case scenario, result in the creator being yeeted into the sun.
Fubar can be a barebones scraper-like app and do nothing out of the box until the user installs 3rd-party extensions.
Or it can make the internet just werk.
Or something in between.
Now is a good time to decide.

**Comment by 93null** [comment]
# On Yeetability
It would be an honor to provide a web experience so pure that I would be yeeted into the sun as a punishment.

# Web 3.0
"Web 3.0" is associated with memes like JPEGs-as-a-service and sudokus-as-a-currency (still better than fiat), but decentralized technologies might be the best way to provide an interface to the web that is friendly to the user.

* Acquire data from peers instead of proprietary sources.
* (presumably) More easily implement cross-platform support.
* Promote freedom.

In summary, this app should probably be built on decentralized technologies to address potential issues.

**Comment by 93null** [comment]
Actually web 3.0 is looking like not what I"m looking for. Guess we're flying into the sun.
